tests/reset_workflow.go

```diff
@@ -651,6 +652,113 @@ func (s *FunctionalSuite) testResetWorkflowReapplyBuffer(
 
 }
 
+func (s *FunctionalSuite) TestUpdateMessageInLastWFT() {
+	s.dynamicConfigOverrides = map[dynamicconfig.Key]interface{}{
+		dynamicconfig.FrontendEnableUpdateWorkflowExecutionAsyncAccepted: true,
+	}
+	tv := testvars.New(s.T().Name())
+	workflowId := "wfid"
+	workflowTypeName := "wftype"
+	taskQueueName := "tq"
+	identity := "worker1"
+	workflowType := &commonpb.WorkflowType{Name: workflowTypeName}
+	taskQueue := &taskqueuepb.TaskQueue{Name: taskQueueName, Kind: enumspb.TASK_QUEUE_KIND_NORMAL}
+	updateId := tv.UpdateID("update-id")
```

Comment:
`update-id` will be already there (it is "type" of the var). Here you should pass a number or don't pass anything if you have just one update in the test.